extensions/blocks/calendly/calendly.php

```diff
@@ -63,7 +62,7 @@ function set_availability() {
 			'missing_plan',
 			array(
 				'required_feature' => 'calendly',
```

Comment:
Maybe we can take that opportunity to update this too?

```suggestion
				'required_feature' => FEATURE_NAME,
```